bobcat/bb-aem-touch-ui/src/main/java/com/cognifide/qa/bb/aem/touch/siteadmin/aem61/SiteadminToolbar.java

```diff
@@ -0,0 +1,191 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.qa.bb.aem.touch.siteadmin.aem61;
+
+import com.cognifide.qa.bb.provider.selenium.BobcatWait;
+import com.cognifide.qa.bb.qualifier.Global;
+import com.cognifide.qa.bb.qualifier.PageObject;
+import com.cognifide.qa.bb.utils.WebElementUtils;
+import com.google.inject.Inject;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+@PageObject
+public class SiteadminToolbar {
+
+  private static final String PUBLISH_LBL = "Publish";
+
+  private static final String UNPUBLISH_LBL = "Unpublish";
+
+  private static final String DELETE_LBL = "Delete";
+
+  private static final String BTN_LIST_SELECTOR = "div.coral-Popover-content > div.endor-List > a";
+
+  @Inject
+  private WebDriver driver;
+
+  @Inject
+  private WebElementUtils webElementUtils;
+
+  @Inject
+  private BobcatWait wait;
+
+  @FindBy(css = "button.cq-siteadmin-admin-actions-edit-activator[title='Open']")
+  private WebElement openButton;
+
+  @FindBy(css = "form.cq-siteadmin-admin-createpage")
+  private CreatePageActions createPageActions;
+
+  @FindBy(css = "button.coral-Button--graniteActionBar.cq-wcm-paste-activator")
+  private WebElement pastePageBtn;
+
+  @FindBy(css = "i.coral-Icon.coral-Icon--delete")
+  @Global
+  private WebElement deleteBtn;
+
+  @Global
+  @FindBy(css = "form.activation-wizard")
+  private CoralWizard activationWizard;
+
+  @Global
+  @FindBy(css = "form.activation-wizard")
+  private ReplicateLaterWindow replicateLaterWindow;
+
+  @Global
+  @FindBy(css = "form.move-page-wizard")
+  private MovePageWizard movePageWizard;
+
+  @Global
+  @FindBy(css = "div.coral-Modal.coral-Modal--notice")
+  private WebElement confirmationModal;
+
+  @Global
+  @FindBy(css = "div.foundation-ui-notification.coral-Alert.coral-Alert--info")
+  private WebElement successInfo;
+
+  public void openPage() {
+    openButton.click();
+  }
+
+  public void publishPage() {
+    getMoreBtn().click();
+    getPublishBtn().click();
+    clickButtonFromMoreSection(PUBLISH_LBL);
+    if (!successInfoAppeared()) {
+      activationWizard.submit();
+    }
+  }
+
+  private boolean successInfoAppeared() {
+    return driver.findElements(
+        By.cssSelector("div.foundation-ui-notification.coral-Alert.coral-Alert--info")).
+        size() > 0;
+  }
+
+  private WebElement getMoreBtn() {
+    return driver.findElement(By.cssSelector("i.coral-Icon.coral-Icon--more"));
+  }
+
+  private WebElement getPublishBtn() {
+    return driver.findElement(By.
+        cssSelector("a.cq-siteadmin-admin-actions-publish-activator[title='Publish']"));
+  }
+
+  private WebElement getDeleteBtn() {
+    if (!deleteBtn.isDisplayed()) {
+      getMoreBtn().click();
+      wait.withTimeout(1);
```

Comment:
Actually the case is about different flows depending on how many pages is selected. When single page is selected, there is no need to click getMoreButton.